test/elixir_analyzer/exercise_test/common_checks/function_annotation_order_test.exs

```diff
@@ -0,0 +1,373 @@
+defmodule ElixirAnalyzer.ExerciseTest.CommonChecks.FunctionAnnotationOrderTest do
```

Comment:
One last thing to make the tests cleaner. Now that we don't have an error with parameters, I think we can switch the style of tests to make them more readable. Could you change them to [look like these](https://github.com/exercism/elixir-analyzer/blob/main/test/elixir_analyzer/exercise_test/common_checks/unless_with_else.exs).

Reply:
Very nice, thank you for the effort!